app/views/companies/_form.html.haml

```diff
@@ -0,0 +1,18 @@
+= form_for @company do |f|
+  - if @company.errors.any?
+    #error_explanation
+      %h2= "#{pluralize(@company.errors.count, "error")} prohibited this company from being saved:"
+      %ul
+        - @company.errors.full_messages.each do |message|
+          %li= message
+
+  .field
+    = f.label :name
+    = f.text_field :name, class: "form-control"
+  .field
+    = f.label :description
+    = f.text_area :description, class: "form-control"
+  .field
+    = f.number_field :user_id, id: :company_user_id, class: "form-control", value: current_user.id, type: :hidden
```

Comment:
don't need this one. We don't want this to be posted from the client side. We will use the current_user in the background.